src/api/public/apidocs-new/paths/person.yaml

```
@@ -38,3 +38,35 @@ get:
       $ref: '../components/responses/unauthorized.yaml'
   tags:
     - Person
+
+post:
+  summary: Executes a command on the person endpoint.
+  description: >-
+    Allows executing command on the person endpoint.
+    As of now, the only command allowed is 'register'.
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: query
+      name: cmd
+      schema:
+        type: string
+      required: true
+      description: The command to execute
+      example: register
+  requestBody:
+    description: Optional description in *Markdown*
+    required: true
+    content:
+      application/xml:
+        schema:
+          $ref: '../components/schemas/unregisteredperson.yaml'
```

```suggestion
          $ref: '../components/schemas/unregistered_person.yaml'
```
Nitpick: adding a `_` will improve readibility.